### ISSUE TITLE
[Security] - On release version { CTRL + SHIFT + I } shortcut is enable

### ISSUE THREAD

**Issue by vheidari** [buttercup/buttercup-desktop#712]
On the release version {CTRL + SHIFT + I} google chrome developer tools [Inspect element] is enable.

**Comment by perry-mitchell** [comment]
This is a very good point. That being said, the feature is still very useful for debugging. I would personally want to improve security but not reduce debugability. Any suggestions? 🙂

**Comment by sallar** [comment]
@perry-mitchell we can maybe put this under a flag? For example if you run buttercup using a command line flag eg. --debug?

**Comment by perry-mitchell** [comment]
@sallar Good idea. I think the issue is that someone could instruct a Buttercup user to enter code in the dev tools, which is definitely a security risk. We should probably do a large console log in there at least to alert users to the importance of not pasting code.

**Comment by sallar** [comment]
@perry-mitchell great idea!
